pkg/api/server/v1alpha2/server.go

```diff
@@ -79,13 +74,17 @@ type LogPluginServer struct {
 
 	IsLogPluginEnabled bool
 	staticLabels       string
-	MaxRetention       time.Duration
 
 	config *config.Config
 	logger *zap.SugaredLogger
 	auth   auth.Checker
 	db     *gorm.DB
 	client *http.Client
+
+	forwarderBufferDuration time.Duration
```

Comment:
If you decide to update the configuration option name as I suggested above, you can change the var name to match (e.g forwarderDelayDuration).

Reply:
I have changed it to forwarderDelayDuration.